lib/msf/core/payload/python/reverse_http.rb

```diff
@@ -24,11 +27,14 @@ def initialize(info = {})
   def generate(opts={})
     ds = opts[:datastore] || datastore
     opts.merge!({
-      host:       ds['LHOST'] || '127.127.127.127',
-      port:       ds['LPORT'],
-      proxy_host: ds['PayloadProxyHost'],
-      proxy_port: ds['PayloadProxyPort'],
-      user_agent: ds['MeterpreterUserAgent']
+      host:           ds['LHOST'] || '127.127.127.127',
+      port:           ds['LPORT'],
+      proxy_host:     ds['PayloadProxyHost'],
+      proxy_port:     ds['PayloadProxyPort'],
+      user_agent:     ds['MeterpreterUserAgent'],
```

Comment:
Sorry if this was a pre-existing issue, is it OK if we rename 'MeterpreterUserAgent' to 'HttpUserAgent' for consistency? You can modify where it's registered to have 'MeterpreterUserAgent' as an alias. Plus, this field gets used by stagers, so not necessarily a Meterpreter-only field anyway.